packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -291,6 +291,7 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
     onDragEnd: _onDragEnd,
   };
 
+  const shouldForceFocusInput = React.useRef<boolean>(false);
```

Comment:
>useRef [](http://example.com/codeflow?start=38&length=6)

SHORT STORY:
You are correct. I was mistaken and this code looks good.

LONG STORY:
Why `useRef`? Why not use `useState`?

I think we just want to store a value, right? As I understand it that's what `useState` is for.

And `useRef` is for keeping a reference to the DOM, which is not what we're doing here.

_(looking up the documentation again)_
**useRef:** "It’s handy for keeping any mutable value around similar to how you’d use instance fields in classes."

I stand corrected. Apparently `useRef` **is** intended to be used for arbitrary values!

Furthermore, `useState` documentation says "It accepts a new state value _and enqueues a re-render of the component."_

We do NOT want to re-render when changing the value, **so you are correct.** `useRef` makes more sense here.